CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 5.1.0 (in progress)
 ================
-* Your contribution here
```

Comment:
Please leave this here (preferably below your entry) as a hint to the next contributor they should add something! 😄